topo_processor/stac/collection.py

```diff
@@ -58,9 +58,9 @@ def delete_temp_dir(self):
     def create_stac(self) -> pystac.Collection:
         stac = pystac.Collection(
             id=str(ulid.ULID()),
-            description=None,
-            license=None,
+            description="STAC Collection Metadata for Historical Imagery",
```

Comment:
I think historical imagery specific stuff should be isolated into historical imagery specific stuff.